SU-SWS/stanford_person
# D8CORE-1736 Migration support for person importer

# READY FOR REVIEW

# Summary
- see https://github.com/SU-SWS/stanford_profile/pull/217
- Migration entity and configuration for CAP importer

# Need Review By (Date)
- 6/11

# Urgency
- medium

# Steps to Test
1. add an environment variable string to local.settings.php. must be 32 characters long: `putenv('STANFORD_ENCRYPT=************************');`
1. Get dependencies
   a. `composer require su-sws/stanford_profile:dev-D8CORE-1736` OR
   b. `composer require SU-SWS/stanford_person:dev-D8CORE-1736 drupal/encrypt drupal/field_encrypt drupal/key drupal/migrate_file drupal/migrate_plus drupal/migrate_tools drupal/real_aes sainsburys/guzzle-oauth2-plugin`
1. depending on your dependency approach either run `drush cim` or `drush en stanford_person_importer`
1. go to `/admin/config/importers/person-importer` and enter credentials and save
1. choose an org code or a workgroup
1. run `drush cron` and verify content has been imported.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
